gradle/publish.gradle

```diff
@@ -73,7 +73,7 @@ task androidSourcesJar(type: Jar) {
     from android.sourceSets.main.kotlin
 }
 
-task androidJavadocsJar(type: Jar, dependsOn: dokkaHtml) {
+task androidKdokJar(type: Jar, dependsOn: kdocJar) {
     classifier = 'javadoc'
-    from dokkaHtml.outputDirectory
+    from kdocJar
```

Comment:
I'm not sure how test it works without creating _snapshot_ 

**cc** @LukasPaczos @Guardiola31337

Reply:
@RingerJK you can for example use the `publishDebugPublicationToMavenLocal` task and browse the files in your `~/.m2/repository/com/mapbox/navigation/core/{version}`, or even load that artifact in 1TAP project (it's already prepared to accept local artifacts) and verify that the docs are showing up.

Reply:
tested 🛳️